app-sso-client/accelerator.yaml

```diff
@@ -0,0 +1,34 @@
+accelerator:
+  options:
+    - name: workloadName
+      inputType: text
+      label: Workload name
+      required: true
+    - name: workloadHost
+      inputType: text
+      label: Workload host name
+      required: true
+    - name: appSsoServerLabel
+      inputType: text
+      label: "AppSSO server label (e.g. 'name: authserver')"
+      required: true
+
+engine:
+  merge:
+    - include: [ "**/client-registration.yaml", "**/client-registration-claim.yaml" ]
+      chain:
+        - type: ReplaceText
```

Comment:
Is there a way to use a YAML oriented recipe here?

In addition to being more "robust", it would have the benefit (I think) to work also if those files pre-exist in the host accelerator: when you say "change yaml key at path x to value y", you don't need to assume that the value you're changing is equal to something you expect.

Reply:
Although it is possible to replace this transformation with a YAML oriented recipes, it would increase number of transformation used as every property would need to be treated individually. As we are in control of these files the current solution is also robust.
If these files already exist in the host accelerator I would consider accelerator author to be responsible to deal with them and touch them at all.

@ericbottard Please resolve and merge the PR